Feature/storybook (#40)
infinum/eightshift-frontend-libs: README.md

# Eightshift Frontend Libs

> This library is intended to work with [Eightshift Libs](https://github.com/infinum/eightshift-libs/) library and [Eightshift Boilerplate](https://github.com/infinum/eightshift-boilerplate) by containing all frontend parts for your project.

Reviewer: > containing all frontend parts 
->

> containing all the frontend parts

## Documentaiton
For the full documentation please check this [link](https://infinum.github.io/eightshift-frontend-libs).

## Prerequisites

> It's recommended that you know how to work with core editor components and if you know some React.js, as this library is focused on working with new editor, and contains custom blocks. If you want to create your own custom blocks, that knowledge will come in handy.

Reviewer: This first sentence seems a bit confusing. I would maybe change it to: 

> It's recommended that you know how to work with core editor components and if you know some React.js, as this library is focused on working with new editor, and contains custom blocks.
->
>The knowledge of the core editor components and React.js is recommended since this library is focused on working with new editor and contains custom blocks.

Reviewer: ```suggestion
It's recommended that you know how to work with core editor components and if you know some React.js, as this library is focused on working with new editor, and contains custom blocks. If you want to create your own custom blocks, that knowledge will come in handy.
```The knowledge of the core editor components and React.js is recommended since this library is focused on working with new editor, and contains custom blocks.

Author: 👍

> It's recommended that you know how to work with core editor components and if you know some React.js, as this library is focused on working with new editor, and contains custom blocks. If you want to create your own custom blocks, that knowledge will come in handy.

Reviewer: ```suggestion
The knowledge of the core editor components and React.js is recommended since this library is focused on working with new editor, and contains custom blocks. If you want to create your own custom blocks, that knowledge will come in handy.
```

Author: 👍

> It's recommended that you know how to work with core editor components and if you know some React.js, as this library is focused on working with new editor, and contains custom blocks. If you want to create your own custom blocks, that knowledge will come in handy.

Author: I have made changes in the documentation

> 

Reviewer 2: The additional notice should be added here that the library can be used for pure webpack/js/scss functionality, but if the blocks are to be used, `eightshift-libs` must be used in tandem.
